src/Orchard.Web/Web.config

```diff
@@ -15,6 +15,12 @@
         <add key="webpages:Version" value="3.0.3"/>
         <add key="log4net.Config" value="Config\log4net.config"/>
         <add key="owin:AppStartup" value="Orchard.Owin.Startup, Orchard.Framework"/>
+    
+<!-- optional additional comma-separated list of folders for Modules or Themes -->
+        <add key="Modules" value="~/Ibn_Modules"/>
+<!--
+        <add key="Themes" value="~/Themes/MoreThemes" />
+-->

```

Comment:
web.config has two benefits:
 1. it is definition and not implementation
 2. web.config is better known and its syntax it more accessible (users do not need to understand the implementation details).

My personal preference is to keep it in web.config.